### ISSUE TITLE
write documentation for this repo

### ISSUE THREAD

**Issue by mlite** [cee-studio/cee-utils#1]
This repo contains code that can be shared. So API documentation makes more sense.

**Comment by mlite** [comment]
let's start with logconf.c

**Comment by mlite** [comment]
Adding usage examples will help understand how to use some features.

**Comment by LucasMull** [comment]
Yes, thats a good idea. We can move tests from orca/test that use any of these files to here, it makes more sense than keeping it there!

**Comment by mlite** [comment]
@LucasMull  This repo will be more important as it's the building block for other projects.  Documentation of json-actor, and other will be more valuable.  Please treat this as the background task, it means we do this whenever there is no other high priority tasks.
